src/pelz-service/main.c

```diff
@@ -148,8 +149,8 @@ int main(int argc, char **argv)
   pelz_service((const int) max_requests, (const int) port_open, (const int) port_attested, secure);
 
   pelz_log(LOG_INFO, "Shutdown Clean-up Start");
-  private_pkey_free(eid, &status);
-  if (status != OK)
+  sgx_status = private_pkey_free(eid, &status);
+  if (sgx_status != SGX_SUCCESS || ret != OK)
```

Comment:
Again, I think this should be checking ```status``` rather than ```ret```.

Reply:
Fixed in ea4e7222.